skstats/hypotests/calculators/asymptotic_calculator.py

```diff
@@ -0,0 +1,337 @@
+from .basecalculator import BaseCalculator
+from ..fit_utils.utils import eval_pdf, array2dataset, pll
```

Comment:
I'm personally not a big fan of composite names when they can be avoided. "fitutils" seems nice to me. (I understand that some long names are better with a "_".)

Reply:
Indeed now that I see it, indeed I don't like it either. I will change it.